tests/k6.js

```diff
@@ -0,0 +1,39 @@
+import http from "k6/http";
+import { check } from "k6";
+
+export let options = {
+  insecureSkipTLSVerify: true,
```

Comment:
Shame `k6` doesn't support custom CA from config.

Reply:
Yeah, it's been raised in their tickets, but the bottom line is that it doesn't affect performance, so they won't fix it. It's not really a concern for us either.